go.mod

```diff
@@ -142,7 +136,6 @@ replace (
 	k8s.io/client-go => k8s.io/client-go v0.22.0
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20210305001622-591a79e4bda7 // Pinned due to kubevirt.io downstream dependencies
```

Comment:
Not something that needs to happen for this PR, but we should do something about this pin since we no longer have the kubevirt.io dependencies.